cdragontoolbox/export.py

```diff
@@ -124,7 +125,7 @@ def __init__(self, storage: Storage, output: str, stored=True):
                 if not versions:
                     break
         else:
-            raise ValueError("versions not found: %r" % versions)
+            raise ValueError(f'versions not found: {repr(versions)}')
```

Comment:
You can use `{value!r}` in format string to format `value` using `repr()`.
(Multiple occurences.)

Reply:
Fixed